OOPBASICSPART1TASK5/src/com/gyk/java/oop/basics/PointApplication.java

```diff
@@ -0,0 +1,40 @@
+package com.gyk.java.oop.basics;
+
+public class PointApplication {
+
+    public static void main(String[] args) {
+        // write your code here
+        MyPoint point1 = new MyPoint(1, 8);
+        MyPoint point2 = new MyPoint(1, 2);
+        MyPoint point3 = new MyPoint();
+        int[] coordinates = new int[2];
+        // You must use a loop to print response from getXY method for first point.
+        coordinates = point1.getXY();
+        for (int i = 0; i < coordinates.length; i++) {
+            System.out.println("coordinates from array: " + coordinates[i]);
+            if (i == 0) {
+                point3.setX(coordinates[i]);
+            } else if (i == 1) {
+                point3.setY(coordinates[i]);
+            }
+        }
+        System.out.println("Coordinates got from loop: " + point3);
+
+        // You must print second point getXY data x and y coordinates without using loop.

```

Comment:
Comments which bring no value should not be included in the code.

Reply:
ok.